spec/serializers/move_serializer_spec.rb

```diff
@@ -152,63 +152,12 @@
 
   describe 'allocation' do
     context 'with an allocation' do
-      let(:adapter_options) do
-        { include: %i[allocation], fields: MoveSerializer::INCLUDED_FIELDS }
-      end
+      let(:adapter_options) { {} }
       let(:move) { create(:move, :with_allocation) }
-      let(:expected_json) do
-        [
-          {
-            id: move.allocation.id,
-            type: 'allocations',
-            attributes: {
-              moves_count: move.allocation.moves_count,
-              created_at: move.allocation.created_at.iso8601,
-            },
-            meta: {
-              moves: {
-                total: 1,
-                filled: 1,
-                unfilled: 0,
-              },
-            },
-            relationships: {
-              from_location: {
-                data: {
-                  id: move.from_location.id,
-                  type: 'locations',
-                },
-              },
-              to_location: {
-                data: {
-                  id: move.to_location.id,
-                  type: 'locations',
-                },
-              },
-            },
-          },
-        ]
-      end
 
       it 'contains an allocation relationship' do
         expect(result_data[:relationships][:allocation]).to eq(data: { id: move.allocation.id, type: 'allocations' })
       end
-
-      it 'contains an included allocation' do
-        expect(result[:included]).to eq(expected_json)
-      end
-    end
-
-    context 'without an allocation' do
-      let(:adapter_options) { { include: MoveSerializer::SUPPORTED_RELATIONSHIPS } }
-
-      it 'contains an empty allocation' do
```

Comment:
can we keep this relationship test for no allocation?

Reply:
I don't think it's very valuable without the included allocation; it's just testing the serializer library behaviour so I cleaned this up.